snippets/csharp/System.Linq/Enumerable/Comparers/CustomComparer.cs

```diff
@@ -190,6 +203,6 @@ lemon 12
 
         //</SequenceEqual>
 
-        Console.ReadLine();
+        _ = Console.ReadLine();
```

Comment:
We've been removing the `Console.ReadLine()` at the end of most samples. That makes it easier to move to the Try.NET experience.